extensions/interactions/base_test.py

```diff
@@ -212,11 +212,17 @@ def test_default_interactions_are_valid(self):
                 feconf.INTERACTIONS_DIR, interaction_id)
             self.assertTrue(os.path.isdir(interaction_dir))
 
-            # In this directory there should only be a config .py file, an
-            # html file, a JS file, a validator.js file,  a directory named
-            # 'static' that contains (at least) a .png thumbnail file,
-            # (optionally) a JS test spec file, (optionally) a JS test spec
-            # file for rules, and (optionally) a protractor.js file.
+            # In this directory there should only the following files:
+            #  Required:
+            #    * A python file called {interactionName}.py.
```

Comment:
Optional: Perhaps interactionName --> InteractionName to emphasize UpperCamelCase.